Testscripts/Linux/perf_ntttcp.sh

```diff
@@ -259,7 +280,10 @@ Run_Ntttcp()
 		Run_SSHCommand "${client}" "${core_mem_set_cmd}"
 	else
 		testType="tcp"
-		echo "test_connections,throughput_in_Gbps,cycles/byte,avglatency_in_us,txpackets_sender,rxpackets_sender,pktsInterrupt_sender" > "${result_file}"
+		# Set sysctl config for creating more than 40960 connections
+		core_mem_set_cmd="sysctl -w kernel.pid_max=122880; sysctl -w vm.max_map_count=655300; sysctl -w net.ipv4.ip_local_port_range='1024 65535'"
```

Comment:
Are those numbers static? No need to be presenting in variables?

Reply:
Yeah, we don't need to presenting in variables.